packages/grid/_modules_/grid/components/columnHeaders/ColumnHeaderSortIcon.tsx

```diff
@@ -30,13 +32,21 @@ export const ColumnHeaderSortIcon = React.memo(function ColumnHeaderSortIcon(
       <div>
         {index != null && (
           <Badge badgeContent={index} color="default">
-            <IconButton aria-label="Sort" title="Sort" size="small">
+            <IconButton
+              aria-label={apiRef!.current.getLocaleText('columnHeaderSortIconLabel')}
+              title={apiRef!.current.getLocaleText('columnHeaderSortIconTitle')}
```

Comment:
Interesting. What's the motivation for using two different keys between title and aria-label? So far, we have considered  the two were identical:

https://github.com/mui-org/material-ui/blob/506181fde08588ee7441a09800e3ed9a218bcba2/packages/material-ui/src/Autocomplete/Autocomplete.js#L437-L438

Reply:
Interesting, I also don't really have a solid argument for them to be separate. I did it that way because it looked like devs will have a more fine grained control over the 2, but now if i think about it making both the title and the label the same does make things easier and it will keep the public API more on point.